packages/eslint-plugin/src/rules/adjacent-overload-signatures.ts

```diff
@@ -21,24 +22,11 @@ type MemberDeclaration =
   | TSESTree.NamedExportDeclarations;
 
 export default createRule({
-  name: 'adjacent-overload-signatures',
-  meta: {
-    type: 'suggestion',
-    docs: {
-      description: 'Require that function overload signatures be consecutive',
-      recommended: 'stylistic',
-    },
-    schema: [],
-    messages: {
-      adjacentSignature: 'All {{name}} signatures should be adjacent.',
-    },
-  },
-  defaultOptions: [],
   create(context) {
```

Comment:
This might be a case where we want to ditch the alpha sorting.
I think that plugin defs are better sorted with the meta-properties being first and `create` being last.